src/test/java/seedu/address/model/fixedexpense/UniqueFixedExpenseListTest.java

```diff
@@ -0,0 +1,214 @@
+package seedu.address.model.fixedexpense;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.fixedexpense.exceptions.DuplicateFixedExpenseException;
+import seedu.address.model.fixedexpense.exceptions.FixedExpenseNotFoundException;
+
+class UniqueFixedExpenseListTest {
+
+    //They have a personBuilder, but since we dont have, improvise by using this for now
```

Comment:
Try not to write comments as conversations or note to self. We will be removing them but let's stick close to the textbook best practices.
```suggestion
    // Temporary fields that will be replaced by FixedExpenses made using a FixedExpenseBuilder.
```